src/map/entities/baseentity.cpp

```diff
@@ -231,3 +231,8 @@ bool CBaseEntity::IsDynamicEntity() const
 {
     return this->targid >= 0x700;
 }
+
+uint8 CBaseEntity::UpdateSpeed(bool run)
```

Comment:
`run` is unused now. Is this intentional?

Reply:
If it is, denote its intentional with

`std::ignore = run`

Reply:
This was only used for battle entities, so I just moved it up and added a type check in pathfind.

Reply:
That works